spec/features/user_show_spec.rb

```diff
@@ -100,5 +100,26 @@
       sleep(1)
       expect(current_path).to eq(user_posts_path(@su))
     end
+
+    it 'should redirect to posts show page when you click on a users post' do
+      visit user_path(@fu)
+      sleep(1)
+      click_link(@su.name)
+      sleep(1)
+      expect(current_path).to eq(posts_path)
+    end
+
+    it 'should redirect to users posts index page when you click on a see all posts.' do
+      visit posts_path
+      sleep(1)
+      click_link('See all posts')
+      sleep(1)
+      expect(current_path).to eq(user_posts_path(@su))
+    end
+    end
```

Comment:
- [ ] After you implemented some changes that the first reviewer suggested, I noticed that 30 tests out of 34 tests are failing. Kindly fix the tests and ensure that all tests pass after implementing all changes. 💻 😎 

![image](https://github.com/sheezu900/blog-rails/assets/77106962/0caaffb6-ec31-4955-85af-079d96c7a1a9)